cmd/broker/update_test.go

```diff
@@ -3106,4 +3106,127 @@ func TestUpdateOIDC(t *testing.T) {
 
 		assert.Len(t, *runtime.Spec.Shoot.Kubernetes.KubeAPIServer.AdditionalOidcConfig, 0)
 	})
+	t.Run("should remove JWKS from OIDC config", func(t *testing.T) {
+		// given
+		cfg := fixConfig()
+		cfg.Broker.EnableJwks = true
+		suite := NewBrokerSuiteTestWithConfig(t, cfg)
+		defer suite.TearDown()
+		iid := uuid.New().String()
+
+		resp := suite.CallAPI("PUT", fmt.Sprintf("oauth/cf-eu10/v2/service_instances/%s?accepts_incomplete=true&plan_id=7d55d31d-35ae-4438-bf13-6ffdfa107d9f&service_id=47c9dcbf-ff30-448e-ab36-d3bad66ba281", iid),
+			`{
+				"service_id": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
+				"plan_id": "5cb3d976-b85c-42ea-a636-79cadda109a9",
+				"context": {
+					"globalaccount_id": "g-account-id",
+					"subaccount_id": "sub-id",
+					"user_id": "john.smith@email.com"
+				},
+				"parameters": {
+					"name": "testing-cluster",
+					"oidc": {
+						"clientID": "id-initial",
+						"signingAlgs": ["PS512"],
+						"issuerURL": "https://issuer.url.com",
+						"encodedJwksArray": "andrcy10b2tlbi1kZGVmYXVsdA=="
```

Comment:
Key is named "encodeJwksArray" but encoded value is simple string (with typo): `jwks-token-ddefault`